docs.md

```diff
@@ -367,6 +367,7 @@ Clicking an unselected checkbox adds its value to this array, and clicking a sel
 *   `input` **[Object][145]** A `redux-form` [input][146] object
 *   `meta` **[Object][145]** A `redux-form` [meta][147] object
 *   `options` **[Array][143]** An array of checkbox values (strings, numbers, or key-value pairs)
+*   `checkboxInputProps` **[Object][145]** An object of key-value pairs representing props to pass down to all checkbox inputs (optional, default `{}`)
```

Comment:
Can you give an example of what the key/value pairs look like, either here or in the example below. I think an example with the `className` key would be most useful.